auth-service/src/main/kotlin/uk/gov/communities/delta/auth/config/LDAPConfig.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.communities.delta.auth.config
+
+class LDAPConfig {
+    companion object {
+        val LDAP_URL = System.getenv("DELTA_LDAP_URL") ?: "ldap://localhost:2389"
```

Comment:
README should mention the need for port forwarding localhost:2389 to say test environment's dluhctest.local